roles/attach-ib/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Task flow inspired by MichaelRigart.interfaces role - thanks markgoddard
+# # Some systems do not support /etc/modules-load.d, in which case we fall back
+# # to /etc/modules.
+- name: Check whether /etc/modules-load.d exists
+  stat:
+    path: /etc/modules-load.d
+  register: modules_load_stat
+
+- name: Ensure any required kernel modules are loaded
+  modprobe:
```

Comment:
Can we not rely on the rdma service to load these?

Reply:
Yes, in fact this was part of the problem when restarted the hosts and they weren't loading IB drivers correctly was because they were present inside modules-probe. Perhaps I will simply check that rdma is running as a service.